frontend/lib/ifixit-api/productData.ts

```diff
@@ -6,6 +6,7 @@ const IFixitFindProductQuerySchema = z.object({
    variantOptions: z.record(z.array(z.string())),
    compatibilityNotes: z.array(z.string()),
    redirectSkuUrl: z.string().nullable(),
+   categories: z.array(z.string()),
 });
```

Comment:
The iFixit API is not currently sending categories. dev_block 👍🏻 because this can't be QA'd

https://github.com/iFixit/ifixit/blob/424bcb4fe701eeb5ed9f85944bd510bc6b99f5cb/iFixit/Store/Controllers/ProductsController.php#L65-L69